test/go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/viper v1.4.0
+	github.com/spf13/viper v1.5.0
```

Comment:
Why update viper here?

Reply:
Looks like viper package got updated. 
The line appears after running 'go build' for all modules, which is one of the check of CI.